client/js/admin.js

```diff
@@ -169,13 +169,14 @@ const attachHandles = function() {
 	$(".copy-path").click(function(e) {
 		e.stopPropagation()
 		const path = $(this).attr("path")
-		const cb = $("#copy-buffer")
-		cb.text(path)
-		const range = document.createRange()
-		range.selectNode(cb[0])
-		window.getSelection().empty ? window.getSelection().empty() : undefined
-		window.getSelection().removeAllRanges ? window.getSelection().removeAllRanges() : undefined
-		window.getSelection().addRange(range)
+		const $copyBuffer = $("#copy-buffer")
```

Comment:
ehhhh, I would rather not have it (perhaps because so many languages use that syntax to denote that something **is** a variable at all). @bluepichu tiebreaker?